internal/formatter/resource_permissions.go

```diff
@@ -0,0 +1,99 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package formatter
+
+import (
+	"fmt"
+	"encoding/json"
+
+	"github.com/sirupsen/logrus"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+const (
+	defaultResourcePermissionListing = "table {{.ResourceName}}\t{{.OperationDescription}}\t{{.ResourceType}}\t{{.OperationType}}"
```

Comment:
This seems misaligned. please do a `go fmt` or use the VSCode Go plugin, as it automatically formats things.

Reply:
Seems to be this way even after go fmt :(
And there was some issue with the plugins- not all of them are installed/working properly i think